templates/cli/lib/commands/deploy.js.twig

```diff
@@ -445,9 +447,7 @@ const deployCollection = async ({ all, yes } = {}) => {
     const collections = [];
 
     if (all) {
-        if (localConfig.getCollections().length === 0) {
-            throw new Error("No collections found in the current directory. Run `{{ language.params.executableName }} init collection` to fetch all your collections.");
-        }
```

Comment:
Let's keep these checks and error messages because it gives some guidance as o how to resolve the problem.

Reply:
Agreed. 
One question here though. The error messages for `deploy function`

`Error: No functions found in the current directory.`

While, for other resources the error message is more descriptive, 
` Error No collections found in the current directory. Run `appwrite init collection` to fetch all your collections`

where it asks to run the `appwrite init [resource]`

Reply:
That's fine. we can leave the function error message as is.